### PR TITLE
Organizes the donator loadout stuff into it's own folder.

### PR DESCRIPTION
**Pull request by theselfish**
## About The Pull Request

I also relocked the pocket watch, and made one that's unbroken.
image
TL:DR Everyone walking around with broken pocket watches set to 7:07pm makes no sense.
It works and looks exactly the same. The difference is the new one can tell time as well. So it's kinda an upgrade.

## Why It's Good For The Game

Joe Hill's famous last words were "Don't waste any time in mourning—organize."
Admittedly he meant something entirely different but, still. It makes everything neater.

## Changelog

:cl:
add: Someone finally repaired the damn pocket watches (may need to check your loadouts)
spellcheck: Made a few loadout item names headline consistent.
code: Moved ckey locked loadout items to their own folder.
/:cl:

### CODE REVIEW

**Comment by theselfish** [comment]
image
Oh and proof the time works.

File: modular_zubbers/code/modules/loadouts/overrides/loadout_items/loadout_datum.dm [hunk 1]
```diff
@@ -69,3 +69,37 @@
 /datum/loadout_item/under/jumpsuit/utility_sec/New()
 	. = ..()
 	LAZYOR(restricted_roles,JOB_SECURITY_MEDIC)
```
Comment (Majkl-J) [inline modular_zubbers/code/modules/loadouts/overrides/loadout_items/loadout_datum.dm:71]: Whilst we're here, you'll make me very happy if you redo these back to
```suggestion
/datum/loadout_item/under/jumpsuit/utility_sec/New()
	. = ..()
	restricted_roles |= JOB_SECURITY_MEDIC
```

because I forgot 3 months ago

Comment (theselfish) [reply modular_zubbers/code/modules/loadouts/overrides/loadout_items/loadout_datum.dm:71]: Sure gimme two minutes.

**Review by projectkepler-ru — APPROVED** [review]

**Review by Majkl-J — APPROVED** [review]
Based
